charts/kubeai/templates/securityContextConstraints.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Capabilities.APIVersions.Has "security.openshift.io/v1" }}
+{{- if .Values.modelServiceAccount.create -}}
```

Comment:
same comment in regards to setting custom service account and create=false

Reply:
Agree